src/plugins/share/public/components/context/index.tsx

```diff
@@ -26,6 +29,10 @@ export interface IShareContext extends ShareContext {
   snapshotShareWarning?: string;
   objectTypeTitle?: string;
   isEmbedded: boolean;
+  theme: ThemeServiceSetup;
+  i18n: I18nStart;
+  intl?: InjectedIntl;
```

Comment:
Would it be possible to not make this optional? I'm guessing it would cause issues in Jest tests, which can be remedied.

```suggestion
  intl: InjectedIntl;
```